CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * **(breaking change)** Removed `ifdef` from the language
 * **(breaking change)** Removed `PointerIO`
 * **(breaking change)** The `body` property of `HTTP::Request` is now an `IO?` (previously it was `String`). Use `request.body.try(&.gets_to_end)`  if you need the entire body as a String.
+* **(breaking change)** `MemoryIO` has been renamed to `IO::Memory`. The old name can still be used but will produce a compile-time warning. `MemoryIO` will be removed immediately after 0.20.0.
```

Comment:
Do you mean removed in 0.21.0 or 0.20.1? This comment seems to imply the latter, which I don't think is what you meant.